plugins/chat/assets/javascripts/discourse/components/chat-channel-settings-view.js

```diff
@@ -61,7 +61,7 @@ export default class ChatChannelSettingsView extends Component {
   @computed("autoJoinAvailable", "togglingChannelWideMentionsAvailable")
   get adminSectionAvailable() {
     return (
-      this.chatGuardian.canEditChatChannel &&
```

Comment:
Can this method be converted into a property? I believe that's how we usually do it.

Reply:
:+1: When I did there was some weird things going on with computed properties...

Anyways, chat-guardian on client side was an experiment and I don't think I will keep it now that we have system tests.